cattle/plugins/docker/compute.py

```diff
@@ -280,67 +421,39 @@ def to_upper_case(key):
                 start_config['binds'] = binds_map
         except (KeyError, AttributeError):
             pass
-
         try:
-            vfcs = instance['dataVolumesFromContainers']
-            container_names = [vfc['uuid'] for vfc in vfcs]
-            if container_names:
-                start_config['volumes_from'] = container_names
-        except KeyError:
-            pass
-
-        try:
-            devices = instance.data.fields['devices']
```

Comment:
Yes, since it's a straight pass-through, it is handled by `START_CONFIG_FIELDS` and `_setup_simple_config_fields`.